presentation/lecture_notes.md

```diff
@@ -0,0 +1,319 @@
+
+# Smart Data Lake Builder Hands-On Training
+> &#x1F4DD; 
+
+## Goal
+* tain loving friend imagine to replace short distant flights with train rides
+* discover/count all flights which 
+  - starting from a certain airport 
+  - <500km
+  - asumme destination airport Bern (LSZB) or Frankfurt (EDDF)
+
+![map about flights around Bern](images/flights_BE.png)
+
+* as data engineers we need to:
+  - download data
+  - combining/filter/transform data in a general manner
+  - store processed data, thus it can be utilized for various use cases
+  - analyse/compute data for a specific application
+
+
+## Why Smart Data Lake?
+* also called Lakehouse
+* combining the flexibility of Data Lakes with the advantages of Data Warehouses
+
+### Data Warehouse
+  - :heavy_plus_sign: preface preparation, structuring and securing data for high quality reporting
+  - :heavy_minus_sign: slow changes, less flexible
+  - :heavy_minus_sign: no horizontal scaling
+  - :heavy_minus_sign: high license cost, expensive
+  - :heavy_minus_sign: no unstructured data processing
+  - :heavy_minus_sign: insufficient integration of AI
+### Data Lake
+  - :heavy_plus_sign: performant and flexible
+  - :heavy_plus_sign: scalable, open source
+  - :heavy_plus_sign: unstructured and complex data structures
+  - :heavy_minus_sign: siloed initatives, redundancy

```

Comment:
How does a Smart Datalake prevent this? We also duplicate data multiple times with all our layers